src/Directory.Build.props

```diff
@@ -5,6 +5,11 @@
   <!-- This file is imported by all projects at the beginning of the project files -->
   <Import Project="$(RepoToolsetDir)Settings.props" />
 
+  <PropertyGroup>
+    <IncludeSymbols>true</IncludeSymbols>
```

Comment:
Sorry, I completely misunderstood this change. For some reason I thought these were the changes to properly create the symbol packages for runtime. So you are right embedded PDBs are just fine for the tools.